src/cmd/vendored.go

```diff
@@ -83,6 +92,50 @@ var zarfCli = &cobra.Command{
 	DisableFlagParsing: true,
 }
 
+func useEmbeddedTofu() error {
+	// Create an executable tofu binary
+	tmpTofuBinary, err := os.CreateTemp(config.CommonOptions.TempDirectory, "tofu")
+	if err != nil {
+		return err
+	}
+	if err = tmpTofuBinary.Chmod(0755); err != nil {
+		return err
+	}
+	defer os.Remove(tmpTofuBinary.Name())
+
+	if _, err = io.Copy(tmpTofuBinary, io.NopCloser(bytes.NewReader(tofuCLI))); err != nil {
+		return err
+	}
+
+	// ignore 'uds', grab the high level tofu command ('apply', 'plan', etc.) and onward from the CLI args
+	os.Args = os.Args[1:]
+	cmd := osExec.Command(tmpTofuBinary.Name(), os.Args...) //nolint:gosec
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+	err = cmd.Run()
+	return err
+}
+
+var planCmd = &cobra.Command{
+	Use:   "plan",
+	Short: lang.CmdBundlePlanShort,
+	// Args:  cobra.MaximumNArgs(0),
+	RunE: func(_ *cobra.Command, _ []string) error {
+		return useEmbeddedTofu()
+	},
+	DisableFlagParsing: true,
```

Comment:
`DisableFlagParsing` messes with my feature flag implementation, but we'll cross that bridge later